### ISSUE TITLE
Total breakage from 1.5.4 update

### ISSUE THREAD

**Issue by treyharris** [quoid/userscripts#727]
I just updated to v1.5.4 from the App Store and it seems to simply not work at all.

Tapping the icon results in:
<img width="360" alt="image" src="https://github.com/user-attachments/assets/aabc6ea4-ae80-48c2-80fa-436f7f9938d9">
and opening the app gives
<img width="720" alt="image" src="https://github.com/user-attachments/assets/c0cde867-a45c-4b49-a82a-c0263a2d37ea">
Note that **it gives its own version as _v0.0.0 (0)_**.

After that, nothing—it remains prompting to open the app. I tried changing the scripts directory with no effect.

So in other words, all my userscripts just totally stopped and I have no workarounds. Please help.

**Comment by quoid** [comment]
Can you give us some more information?

- What os are your running? 
- What kind of device?
- What was the prior version installed?
